src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -3,266 +3,260 @@
-            display: table-cell;
-        }
-        .datagrid-column-separator {
-            display: none;
-        }
-        .datagrid-placeholder-container {
-            display: none;
-        }
-        .datagrid-fixed-column {
-            display: none;
-        }
-    }
+                &:focus {
+                    outline: 0;
+                }
 
-    // TODO: the nesting in this SCSS is a bit extremes. is it necessary?
-    .datagrid {
-        flex: 1 1 auto;
-        width: auto; // override the basic table's style of width: 100%
-        max-width: none; // override the basic table's style of max-width: 100%
+                &.disabled,
+                &:disabled {
+                    cursor: not-allowed;
+                    opacity: 0.4;
+                    user-select: none;
 
-        .datagrid-body {
-            .datagrid-row:last-child {
-                .datagrid-cell {
-                    &:first-child {
-                        border-radius: 0;
+                    &:hover {
+                        background: none;
                     }
-                    &:last-child {
-                        border-radius: 0;
+
+                    &:active,
+                    &:focus {
+                        background: none;
+                        box-shadow: none;
                     }
                 }
+
+                clr-icon {
+                    vertical-align: middle;
+                    transform: translate3d(0px, -1px, 0);
+                }
             }
         }
 
-        .datagrid-column, .datagrid-cell {
-            text-align: left;
-            min-width: 4rem;
-            &.datagrid-fixed-column {
-                flex: 0 0 $clr-datagrid-fixed-column-size;
-                // IE10 needs this
-                max-width: $clr-datagrid-fixed-column-size;
-                min-width: $clr-datagrid-fixed-column-size;
-            }
+        .datagrid-row-detail-wrapper {
+            flex-basis: 100%;
         }
 
-        .datagrid-head {
-            background-color: $clr-thead-bgcolor;
-            border-bottom: (2 * $clr-table-borderwidth) solid $clr-table-header-border-bottom-color;
+        .datagrid-row-detail {
+            &.datagrid-container {
+                padding-top: 0;
+            }
 
-            .datagrid-row-actions {
-                @extend .datagrid-column;
+            .datagrid-cell {
+                padding-top: 0;
             }
         }
 
-        .datagrid-column {
-            vertical-align: top;
-            background: none;
-            border-bottom: 0;
+        .datagrid-select {
+            min-width: calc(0.375rem - #{$clr-table-borderwidth}); // This fixed an issue that made the cell to wide.
+            .radio label,
+            .checkbox label {
+                display: block;
+                // Override default checkbox styles for width/height
+                min-height: $clr-table-lineheight - $clr-rem-1px;
```

Comment:
nit: space between `$clr-rem - 1px;`

Reply:
Didn't see this when I made changes for the review. Perhaps when I squashed it was updated?